src/formats/vga_image.c

```diff
@@ -1,11 +1,19 @@
+#if !defined(N64_BUILD) && !defined(MIN_BUILD)
 #include <png.h>
+#endif
```

Comment:
Hum. Could we move the png reading from this file to utils/png_reader.h ? Then we could import the png headers etc. there, and also do the png support enable/disable there.

Then the functions, e.g. png_write_paletted() could just return "OK" if there was no support :)

We could also support the png files completely in cmake, then ? Probably.

Reply:
It is okay if that becomes a tracked issue for now?
I don't want to get into anything cmake for now.

Reply:
I don't think we need to do the cmake thing now, but I would definitely still move the ifdeffing to the separate png-related files.